src/main/java/com/google/flourbot/entity/action/SheetAppendAction.java

```diff
@@ -1,36 +1,32 @@
 package com.google.flourbot.entity.action;
 
-import java.util.Map;
-import com.google.flourbot.entity.action.Action;
-
-
 public final class SheetAppendAction extends Action {
 
-    private final ActionType actionType = ActionType.SHEET_APPEND;
+  private final ActionType actionType = ActionType.SHEET_APPEND;
```

Comment:
have you started using some linter or are you cleaning up stuff manually? If you have a linter great, if you do it by hand - don't bother, we can do it later ;)

Reply:
Yes it's automatic with google-java-formatter :)